crates/worker/src/partition/state_machine/effects.rs

```diff
@@ -532,10 +529,11 @@ impl Effect {
                         span_context.is_sampled(),
                         span_context.as_parent(),
                         "background_invoke",
-                        otel.name = format!("background_invoke {service_method}"),
-                        rpc.service = %full_invocation_id.service_id.service_name,
-                        rpc.method = %service_method,
-                        restate.invocation.id = %full_invocation_id,
+                        otel.name = format!("background_invoke {invocation_target}"),
+                        rpc.service = %invocation_target.service_name(),
+                        rpc.method = %invocation_target.handler_name(),
+                        restate.invocation.id = %invocation_id,
+                        restate.invocation.target = %invocation_target,
```

Comment:
Is `restate.invocation.target` enough (removing `rpc.service` and `rpc.method`)?

Reply:
no, see comment above.